### PR TITLE
Activation epochs endpoint

### PR DESCRIPTION
**Pull request by mateigraura**
Add new endpoint `/network/enable-epochs` in order to retrieve activation epochs metric from observers

Testing procedure:
Send a GET request to the proxy on the endpoint `{proxy}/v1.0/network/enable-epochs`

### CODE REVIEW

File: cmd/proxy/config/apiConfig/v1_0.toml [hunk 1]
```diff
@@ -46,7 +46,8 @@ Routes = [
     { Name = "/status/:shard", Open = true, Secured = false, RateLimit = 0 },
     { Name = "/economics", Open = true, Secured = false, RateLimit = 0 },
     { Name = "/config", Open = true, Secured = false, RateLimit = 0 },
-    { Name = "/esdts", Open = true, Secured = false, RateLimit = 0 }
+    { Name = "/esdts", Open = true, Secured = false, RateLimit = 0 },
+    { Name = "/enable-epochs", Open = true, Secured = false, RateLimit = 0 }
```
Comment (bogdan-rosianu) [inline cmd/proxy/config/apiConfig/v1_0.toml:50]: add also in `v_next`

File: process/nodeStatusProcessor.go [hunk 2]
```diff
@@ -102,6 +105,29 @@ func (nsp *NodeStatusProcessor) GetNetworkConfigMetrics() (*data.GenericAPIRespo
 	return nil, ErrSendingRequest
 }
 
+// GetEnableEpochsMetrics will simply forward the activation epochs config metrics from an observer in the given shard
```
Comment (bogdan-rosianu) [inline process/nodeStatusProcessor.go:108]: drop the `in the given shard`

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: cmd/proxy/config/apiConfig/v1_0.toml (hunk 1 above)
Comment (mateigraura) [reply cmd/proxy/config/apiConfig/v1_0.toml:50]: done

File: process/nodeStatusProcessor.go (hunk 2 above)
Comment (mateigraura) [reply process/nodeStatusProcessor.go:108]: done

**Review by bogdan-rosianu — DISMISSED** [review]

File: process/nodeStatusProcessor.go [hunk 3]
```diff
@@ -24,6 +24,9 @@ const NodeStatusPath = "/node/status"
 // NodeStatusPath represents the path where an observer exposes all the issued ESDTs
```
Comment (miiu96) [inline process/nodeStatusProcessor.go:24]: Add this API route also in README.md

Comment (mateigraura) [reply process/nodeStatusProcessor.go:24]: done

**Review by miiu96 — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed

**Pull request merged**